src/message_handler.cc

```diff
@@ -228,7 +227,8 @@ void MessageHandler::Run(InMessage &msg) {
       } catch (NotIndexed &) {
         throw;
       } catch (...) {
-        reply.Error(ErrorCode::InternalError, "failed to process " + msg.method);
+        reply.Error(ErrorCode::InternalError,
```

Comment:
These intermixed formatting is not very necessary..

Reply:
The project isn't formatted properly therefore the changes. I can try to remove formatting from this PR and make another one with pure clang-format run on src directory. Would that be okay?